src/patterns/fundamentals/layout/stack-split.hbs

```diff
@@ -0,0 +1,10 @@
+---
+name: Stack-Split Object
+notes: The Stack Split pattern is used for keeping items stacked until theres enough room for them to be
+  side by side.
+order: 6
+---
+<div class="StackSplit">
+  <div class="StackSplit__item"><button class="sprk-c-Button">Button</button></div>
```

Comment:
Good idea to use the button and link example here. Thats what people want most from this pattern it seems.